shimmingtoolbox/masking/threshold.py

```diff
@@ -25,4 +16,4 @@ def threshold(data, thr=30):
     Returns:
         numpy.ndarray: Boolean mask with same dimensions as data
     """
-    return data > thr
+    return abs(data) > thr
```

Comment:
As discussed, this might lead to unwanted results if there are negative inputs. (I would expect -35 to be False).

Reply:
Should be fixed by [3cf8dd4](https://github.com/shimming-toolbox/shimming-toolbox/pull/299/commits/3cf8dd4def3aa4239ca5b838a2ea95f7a3f3923f)

Reply:
Idea: simply check if data are complex instead of passing an argument